src/components/Organization/OrgUsersCard/addOrgUserModal.js

```diff
@@ -65,105 +73,90 @@ const AddOrgUserModal = ({ currentOrgHandle }) => {
+      _.findIndex(currentOrgUsers, (user) => user.handle === handle) !== -1
     ) {
-      form.resetFields(["handle"]);
-      return form.setFields([
-        {
-          name: "handle",
-          errors: [
-            `The user [${values.handle}] is already in the organization [${currentOrgHandle}]`
-          ]
-        }
-      ]);
+      setHandle('');
+      setHandleValidateError(true);
+      setHandleValidateErrorMessage(
+        `The user ${handle} is already in the organization ${currentOrgHandle}`
+      );
     } else {
       await addOrgUser({
         org_handle: currentOrgHandle,
-        permissions: parseInt(selected.split("_")[1]),
-        handle: values.handle
+        permissions: parseInt(selected.split('_')[1]),
+        handle: handle,
       })(firestore, dispatch);
     }
   };
 
-  const handlePermissionChange = ({ key }) => {
-    setSelected(key);
-  };
-
-  const permissionLevelsButton = ({ selected }) => {
-    return (
-      <Menu
-        onClick={e => handlePermissionChange({ ...e })}
-        selectedKeys={selected}
-      >
-        <Menu.Item key={"perm_0"}>
-          <EyeOutlined /> Reviewer
-        </Menu.Item>
-        <Menu.Item key={"perm_1"}>
-          <EditOutlined /> Editor
-        </Menu.Item>
-        <Menu.Item key={"perm_2"}>
-          <SafetyOutlined /> Admin
-        </Menu.Item>
-      </Menu>
-    );
+  const handlePermissionChange = (event) => {
+    setSelected(event.target.value);
   };
 
   return (
-    <Form form={form} name="add-org-user" onFinish={onFinish}>
-      <Form.Item
-        name="handle"
-        rules={[
-          {
-            required: true,
-            message: "Please input the user handle you want to add"
-          }
-        ]}
+    <Grid container item={true}>
+      <TextField
+        label="User Handle"
+        variant="outlined"
+        fullWidth
+        value={handle}
+        error={handleValidateError}
+        onChange={onChangeHandle}
+        helperText={handleValidateError ? handleValidateErrorMessage : null}
+      />
+      <Grid container justify="flex-end">
+        <div style={{ padding: '10px' }}>
+          <span style={{ paddingRight: '10px' }}>Select user role</span>
+          <Select value={selected} onChange={handlePermissionChange}>
+            {options.map((option, index) => (
+              <MenuItem key={index} value={option.value}>
+                <div
+                  style={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    justifyContent: 'space-between',
+                  }}
+                >
+                  {option.icon}
+                  <div style={{ paddingLeft: '5px' }}>{option.name}</div>
+                </div>
+              </MenuItem>
+            ))}
+          </Select>
+        </div>
+      </Grid>
+
+      <Button
+        style={{ backgroundColor: '#0f7029' }}
```

Comment:
I think white text would be more readable

Reply:
will do